docs/self-managed/modeler/web-modeler/configuration.md

```diff
@@ -0,0 +1,121 @@
+---
+id: configuration
+title: Configuration
+description: "Read details on the configuration variables of Web Modeler Self-Managed, including components such as REST API, Identity, Keycloak, webapp, and WebSocket."
+---
+
+:::caution Beta Offering
+Web Modeler Self-Managed is currently offered as a [beta release](../../../../reference/early-access#beta)
+with limited availability for enterprise customers only. It is not recommended for production use and there is no maintenance service guaranteed.
+Special [terms & conditions](https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-self-managed/) apply.
+However, we encourage you to provide feedback via your designated support channel or the [Camunda Forum](https://forum.camunda.io/).
+:::
+
+The different components of Web Modeler Self-Managed can be configured using environment variables. Each component's variables are described below.
+For a working example configuration, that shows how the components are correctly wired together, see the [Docker Compose file for Web Modeler](../../../platform-deployment/docker#web-modeler-1).
+
+### Configuration of the `restapi` component
+
+#### Database
+
+Web Modeler requires a PostgreSQL database as persistent data storage (other database systems are currently not supported.)
+
+| Environment variable  | Description            | Example value          |
+| --------------------- | ---------------------- | ---------------------- |
+| `RESTAPI_DB_HOST`     | Database host name     | `postgres.example.com` |
+| `RESTAPI_DB_PORT`     | Database port          | `5432`                 |
+| `RESTAPI_DB_NAME`     | Database name          | `modeler-db`           |
+| `RESTAPI_DB_USER`     | Database user name     | `modeler-user`         |
+| `RESTAPI_DB_PASSWORD` | Database user password | \*\*\*                 |
+
+#### SMTP / email
+
+Web Modeler requires an SMTP server to send notification emails to users.
```

Comment:
@christinaausley: I changed this back to "a**n** SMTP" server. You use "an" if the next word's pronunciation starts with a vowel sound (in this case "ess"), don't you?

Reply:
You are correct here 👍